bciers/apps/administration/tests/components/operations/OperationInformationForm.test.tsx

```diff
@@ -55,16 +55,16 @@ describe("the OperationInformationForm component", () => {
     expect(screen.getByText(/Operation Type/i)).toBeVisible();
   });
 
-  it("should have unchecked task list sections when no formData is provided", async () => {
-    render(<OperationInformationForm formData={{}} schema={testSchema} />);
+  // it("should have unchecked task list sections when no formData is provided", async () => {
```

Comment:
Is this meant to be commented out or deleted?

Reply:
@BCerki I am unsure as this is not part of my changes.